content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+
+Electronic health records are extremely useful in providing information about epidemics in real time. A patient may interact with the healthcare system days before receiving a test (if any), and several more days before receiving a positive result and being reported as a confirmed case by a state's Department of Health. This can make our signal an **early indicator** of confirmed cases, and certainly an early indicator of deaths. Early knowledge of an increase in people with COVID-like symptoms is extremely important, as early containment can prevent an outbreak.
+
+Additionally, the reported case counts have other issues:
+
+* Limited testing capacity: Especially prevalent in the earlier days of the pandemic, limited testing capacity means that there were many people who saw a doctor for COVID-like symptoms but never even received a test. These people would be present in our data but absent from the confirmed counts.
+* Backlogs: A patient can be first reported as a confirmed case can be several days after they were tested. Since our data is linked to the date that the patient interacted with the healthcare system, we report a more accurate date.
+* Revisions: Some states have changed their definition of which tests indicate a confirmed COVID-19 case. This can be done retroactively, including hundreds or thousands of past cases and reporting them as "new" on the day on which they changed their definition. These cases are clearly assigned the wrong date, but we have no way of determining when the test was actually taken. Our data is subject to revisions due to lags in reporting, but the revisions are understood and are assigned the correct date.
+
+### Doctor Visits Indicator
+The Doctor Visits indicator is based solely on insurance claims. For a given geographic area and a given day, we receive the count of claims that fall into each of five categories:
+
+1. Total: All claims
+2. Flu: ICD-10 primary code starting with J09, J10, or J11. This is an explicit diagnosis of flu.
+3. COVID-like: ICD-10 primary code in {U07.1, U07.2, B97.29, J12.81, Z03.818, B34.2, J12.89}. Some of these codes correspond to an explicit diagnosis of COVID-19, while others indicate similar symptoms.
+4. Flu-like: ICD-10 primary code of J22 (acute lower respiratory infection) or B34.9 (viral infection, unspecified).
+5. Mixed: ICD-10 primary code of Z20.828 (suspected exposure to COVID-19) or J12.9 (viral pneumonia).
+
+
+We calculate the percent of COVID-like illness (%CLI) as (COVID-like + Flu-like + Mixed - Flu) / Total. We subtract the Flu counts because they are unrelated to coronavirus. The indicator is available at four geographic levels: county, MSA, HRR, and state. See [here](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html) for more details.
```

Comment:
```suggestion
We estimate the percent of COVID-like illness (%CLI) as (COVID-like + Flu-like + Mixed - Flu) / Total. We subtract the Flu counts because, although they are symptomatically COVID-like, they are known to beunrelated to coronavirus, thus making the signal more directly related to COVID-19. The indicator is available at four geographic levels: county, MSA, HRR, and state. See [here](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html) for more details.
```

@mariajahja Did we ever resolved why we are subtracting 'flu'?  Given that on any date, any patient is recorded in at most one of these four categories, it would seem that we need to exclude, but not subtract, the confirmed-flu cases.  And yet maybe subtraction led to better correlations?

Reply:
We originally thought to construct the signal on the "rate" scale rather than "counts". Therefore, we were subtracting the true flu rate from the mixed+flu_like rate, where the latter was thought to have counted patients that had flu but weren't definitive. But yes, the way these counts were constructed should ensure that this doesn't happen. The implementation does work on the (smoothed and padded) rate scale, rather than counts. 

Note that during the summer, the 'flu' category is essentially nil, and so this subtraction didn't do anything. Earlier on in March with a non-zero flu signal, the subtraction looked necessary to avoid having a strong corona signal that we thought was impossible. I assume that coding practices have changed since then, with doctors able to discern more clearly between covid and flu, and so for the upcoming flu season it's not clear to me whether this subtraction is helpful or necessary.

Reply:
Thanks @mariajahja .
> the subtraction looked necessary to avoid having a strong corona signal that we thought was impossible.
This suggests to me a (admittedly post hoc) explanation:  the more confirmed-flu there is, the higher the expected fraction of flu-like that is due to flu.  If true, then rather than subtracting, we should train a sensor on these 4 covariates, and make it an adaptive sensor so it continuously adapts to the most recent coding practices.   @rumackaaron @ryantibs